controllers/usersignup/usersignup_controller_test.go

```diff
@@ -2490,6 +2473,11 @@ func TestUserSignupReactivateAfterDeactivated(t *testing.T) {
 		Spec: toolchainv1alpha1.UserSignupSpec{
 			Userid:   "UserID123",
 			Username: meta.Name,
+			IdentityClaims: toolchainv1alpha1.IdentityClaimsEmbedded{
+				PropagatedClaims: toolchainv1alpha1.PropagatedClaims{
+					//Email: "john.doe@redhat.com",
+				},
+			},
```

Comment:
Do you need this empty IdentityClaims here?

Reply:
We do need to set the email address here otherwise all the metrics tests fail, although I still haven't been able to work out why this is the case.  I've uncommented this line, I didn't intend to commit this change.